### ISSUE TITLE
Table input mapping performance

### ISSUE THREAD

**Issue by Halama** [keboola/docker-bundle#65]
Tables should be exported gziped and than ungziped by docker bundle.
https://github.com/keboola/docker-bundle/blob/2d9c7b42801f39c5cb75e029524fe06cd12ff00b/Docker/StorageApi/Reader.php#L204

**Comment by ondrejhlavacek** [comment]
@Halama jestli spíš nedat defaultně gzip na true v https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/TableExporter.php#L45

tady se to stejně gunzipne - https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/TableExporter.php#L143, takže by to mělo bejt transparentní?

**Comment by ondrejhlavacek** [comment]
No jenže tady se to zase zpátky zagzipuje - https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/TableExporter.php#L156

A docker-bundle to zas bude znova rozgzipovávat. To je taky slušnej overhead. Co kdyby TableExporter dodal vždycky rozgzipovanej soubor a forcoval by gzipování přenosu (pokaždý nastavil gzip na true v exportOptions)?

**Comment by Halama** [comment]
ted jsme na to zas narazili. Ok udělal bych to tak jak píšeš.

**Comment by Halama** [comment]
jenom přemýšlám co když to někdo bude chtít stáhnout gzipnutý.

**Comment by ondrejhlavacek** [comment]
Jo, udělal bych to teda tak, že `exportOptions` metody `TableExporter::exportTable` budou ovlivňovat výstupní formát, nikoliv transfer/export. Export bude vždy gzip, nenapadá mě žádnej důvod, proč by neměl bejt gzip. A pak se podle požadavku v `exportOptions` se na konci spojenej soubor zagzipuje nebo se nechá rozgzipovanej. 

Souhlas?

**Comment by odinuv** [comment]
jo

**Comment by ondrejhlavacek** [comment]
nejsem si jistej, jestli se to děje, přesouvám do správnýho repa
